src/Dfc.CourseDirectory.Database/Pttcd/Tables/TLevelDefinitions.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [Pttcd].[TLevelDefinitions] (
+    [TLevelDefinitionId] UNIQUEIDENTIFIER NOT NULL,
+    [FrameworkCode]      INT              NOT NULL,
```

Comment:
If `FrameworkCode` and `ProgType` pairs unique a constraint to enforce it would be good here

Reply:
I don't see the value of doing that at this time, especially as we're currently controlling data ingress via script currently.